test/specs/rules/disallow-space-after-keywords.js

```diff
@@ -1,7 +1,7 @@
 var Checker = require('../../../lib/checker');
 var assert = require('assert');
 
-describe('rules/disallow-space-after-keywords', function() {
+describe.only('rules/disallow-space-after-keywords', function() {
```

Comment:
By the way, perhaps we should disallow this identifier name... If only we knew of a linter that provided this functionality.... ;-)